### ISSUE TITLE
Some kind of performance loss going from 7.5 to 7.6

### ISSUE THREAD

**Issue by Org-gith** [renpy/renpy#4703]
This doesn't seem to be a cache issue as I followed the recommended guideline of increasing it by 33% and monitored the Vram usage and it was about the same.
This is a video trying to showcase the slow down, it's hard to capture it on video with that kind of bitrate, but I think it translates okay when slowed down at about 0.25/0.5 speed (there is a menu to choose the speed when right-clicking on the video). 

https://streamable.com/t15axd
EDIT: on the leftside it's 7.5, on the right side it's 7.6

In the first part of the video, I'm trying to show how the dissolve to black is laggier on 7.6 compared to 7.5 in the project (with a hiccup at the end when switching between rooms). 
`transitionpointandclickopen = { "master" : Dissolve(0.2) }`

In the second part, I'm trying to show that the transform that makes the text window slide down starts with a hiccup in 7.6, and is less smooth.
```
transform hide_textbox_2:
    yoffset 0
    easeout 0.490 yoffset 452
```

Tried both GL2 and GL1, and the results were the same
I'm not sure if it's a performance loss or something wrong with the atl or dissolves specifically, or something else. In general, the game felt less responsive but this was impossible to really capture that on video, and could be a placebo effect induced by the above, not sure.

I'll keep investigating this and try different things in the coming days, to see if I can find something.

**Comment by renpytom** [comment]
I don't see it, but you can hit f3 to display the fps counter. Put it in performance mode, and it should make drops in framerate obvious. Note that not every change in performance is an issue, as sometimes it's slower in one place and faster in another.

**Comment by Org-gith** [comment]
Thanks, I'll inspect it more closely, it makes sense that performance varies, just I think this one is too noticeable for some reason, hard to close an eye for it from my perspective at least (the video doesn't do it justice, it feels like a mini freeze). It is a bit bothersome but I'll close the issue if I can't find anything. It's hard to pinpoint what would be the culprit and could end up being a big waste of time so I'll try to see if I can find anything first. (this week)

**Comment by Org-gith** [comment]
Wanted to update on this, I think it was my installation gone wrong or Windows having a problem. The first 7.6 install I made was not able to save persistence for some reason, it would ask me to choose a language on startup every time, it had that issue explained and exposed here too. I redid an install with 7.6.1 and even 7.6.0 and the problem seems gone, and it's not asking me to choose the language every time I start it. Seems like it was some kind of problem external to ren'py.
I'll reopen the issue if it shows up again or if it seems to be an actual issue of ren'py, but all seems good I think, sorry for the trouble.

**Comment by Org-gith** [comment]
There is still something slightly off for some reason.
In the project, the window slides up and down after talking, on 7.5 the animation is butter smooth, on 7.6 it always starts with a mini hiccup, 
It's sort of skipping a frame or two. I have picked a moment in the game where it's just a background and the windows sliding up and down, launched the game on the two versions, going back and forth and it consistently skips a bit of the beginning on 7.6 whereas 7.5, it's butter smooth. I'm not sure where it's going on but I'm gonna try the much older nightly builds, see if I can find something, a breaking point or something. The animation also plays a sound so I thought the new more advanced audio could create a slight hiccup but it doesn't appear to be that, even when disabling the sound, the hiccup is still there.

**Comment by Org-gith** [comment]
6 March nightly build didn't do it, and the 13 March nightly build had it.
I tried some of the commits and if I undo this one: https://github.com/renpy/renpy/commit/cd8b496876b8c7706404c431eb3dceffb88ebb8f#diff-501f89a264226a42fca5b308226a347b538c5e98fbdc380cb9aa8876e8c7d217

The problem disappears, I thought it was silly so I tried going back and forth, putting the commit back and forth (the files), but it's consistent, the hiccup just disappears every time I undo this commit, and when I put it back on it reappears. 
Does this seem like something possible? I'll run without it for a while to see if it doesn't come back.
